### PR TITLE
Add ucrt for windows

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge/staged-recipes, @xhochy, this is ready fore review. This is needed for https://github.com/conda-forge/vc-feedstock/pull/37

File: recipes/ucrt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  script:
+    - mkdir -p $PREFIX/Library/bin
+    - 7z x 20348.1.210507-1500.fe_release_WindowsSDK.iso -aoa
+    - msiextract -C tmp "Installers/Universal CRT Redistributable-x86_en-us.msi"
+    - cp "tmp/Program Files/Windows Kits/10/Redist/{{ version }}/ucrt/DLLs/x64/"* $PREFIX
+    - cp "tmp/Program Files/Windows Kits/10/Redist/{{ version }}/ucrt/DLLs/x64/"* $PREFIX/Library/bin
+
+  # This is supposed to be a win-64 package only, but it is noarch: generic because
+  # I don't want to write this in cmd.
+  noarch: generic
```
Comment (xhochy) [inline recipes/ucrt/meta.yaml:23]: Happy to do this for you. With the above commands, I should be able to write this with CMD and Windows compatible dependencies.

Also happy to be added as a maintainer.

Comment (isuruf) [reply recipes/ucrt/meta.yaml:23]: Thanks @xhochy

Comment (isuruf) [reply recipes/ucrt/meta.yaml:23]: Feel free to push to this branch.

**Pull request merged**